spec/features/edit_training_details_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "edit training details" do
+  scenario "edit with valid parameters" do
+    given_a_trainee_exists
+    when_i_visit_the_training_details_page
+    and_i_enter_valid_parameters
+    then_i_am_redirected_to_the_summary_page
+  end
+
+  def given_a_trainee_exists
+    @trainee = create(:trainee)
+  end
+
+  def when_i_visit_the_training_details_page
+    visit "/trainees/#{@trainee.id}/training-details"
```

Comment:
I think it would be more idiomatic to load the url using the page object you created. ie: https://github.com/DFE-Digital/register-trainee-teacher-data/blob/e1e292500a2681665d9310e6a1133d8f73165366/spec/features/trainees/trainee_summary_spec.rb#L20

Reply:
done